src/main/java/com/uber/cadence/internal/replay/DecisionContext.java

```diff
@@ -80,6 +80,9 @@
    */
   SearchAttributes getSearchAttributes();
 
+  /** Propagates the context */
+  void propagateContext();
+
```

Comment:
This API should be more like getCurrentContext() as the DecisionContext API is asynchronous and shoudln't rely on thread locals.